test/test_optimus_checker.ipynb

```diff
@@ -0,0 +1,898 @@
+{
```

Comment:
what is this file doing and where does it get used (sorry lack of context on my part)

Reply:
Just a file used to run some workflows; it uses an ipython notebook with the cromwell-manager package to start/stop/debug.